src/content/docs/en/guides/assets.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: Assets (Experimental)
+description: Learn how to enable experimental asset support in Astro.
+i18nReady: false
+---
+
+**Built-in Asset Support** is enabled in Astro behind an experimental flag. This built-in feature will eventually replace the optional `@astrojs/image` integration.
+
+The new assets experience currently features:
+
+- A new built-in `<Image />` component
+- Relative images with automatic optimization in Markdown and MDX files
+- Integration with content collections
+- Improved error messages and types
+
+:::caution
+Assets are an experimental Astro feature. This API is subject to change before it is marked as stable.
+:::
+
+## Enabling Assets in your Project
+
+Enabling assets may cause some breaking changes in your project. It may also require some manual changes to take advantage of new features.
+
+Please check every section below to avoid errors and to get the most out of using the experimental assets option!
+
+### Update Config
+
+To enable built-in asset support, add the following lines to your `astro.config.mjs` configuration file:
+
+```js title="astro.config.mjs" ins={4-6}
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+ experimental: {
+  assets: true
+ }
+});
+```
+
+### Move your images to `src/assets/`
+
+Create an `src/assets/` folder, which Astro will recognize as your new assets folder.
+
+We **strongly recommend** storing all images to be optimized in the `src/assets/` directory. This location is optional for now, but may become the required location for all asset types in the future.
+
+Your images stored in `src/assets/` can be used by components (`.astro`, `.mdx`, and other UI frameworks) and in Markdown files.
+
+### Update existing `<img>` tags
+
+Previously, importing an image would return a simple `string` with the path of the image. With the new `image` features enabled, imported image assets now match the following signature:
+
+```ts
+interface ImageAsset {
+  src: string;
+  width: number;
+  height: number;
+  format: string;
+}
+```
+
+You must update the `src` attribute of any existing `<img>` tags, and you may also update other attributes that are now available to you from the imported image.
+
+```astro title="src/components/MyComponent.astro" ".src" ".width" ".height" del={2,5} ins={3,6}
+---
+import rocket from '../images/rocket.svg';
+import rocket from '../assets/images/rocket.svg'
+---
+<img src={rocket} width="250" height="250" alt="A rocketship in space." />
+<img src={rocket.src} width={rocket.width} height={rocket.height} alt="A rocketship in space." />
+```
+
+### Update your Markdown files
+
+Relative images can now be referenced in Markdown and MDX files. These will automatically be optimized and hashed by Astro's build process.
```

Comment:
This is only true in Markdown, not MDX at current time (because Erika didn't even know you could use `![]` in MDX). The current happy path for MDX is to import the images as in Astro files.